pkg/koordlet/util/system/cgroup.go

```diff
@@ -70,6 +70,11 @@ func (m *MemoryStatRaw) Usage() int64 {
 	return m.InactiveAnon + m.ActiveAnon + m.Unevictable
 }
 
+func (m *MemoryStatRaw) MemWithPageCacheUsage() int64 {
```

Comment:
`UsageWithPageCache()` may be better
And do we need to count the shared memory for a conservative overcommitment?